src/workloads/setup.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+
+# HISTORY.rst entry.
+VERSION = '0.1.0-preview'
```

Comment:
Will you want it be released as `preview` as I saw the swagger used is GA?

Reply:
Yes swagger is GA but we want to release the CLI client in Preview to get initial customer feedback before removing preview tag.